tests/ignite/contrib/metrics/regression/test_canberra_metric.py

```diff
@@ -63,6 +64,42 @@ def test_compute():
     assert canberra.pairwise([v1, v2])[0][1] == pytest.approx(np_sum)
 
 
+def test_integration_binary_input_with_output_transform():
+    def _test(y_pred, y, batch_size):
+        def update_fn(engine, batch):
+            idx = (engine.state.iteration - 1) * batch_size
+            y_true_batch = np_y[idx : idx + batch_size]
+            y_pred_batch = np_y_pred[idx : idx + batch_size]
+            return idx, torch.from_numpy(y_pred_batch), torch.from_numpy(y_true_batch)
+
+        engine = Engine(update_fn)
+
+        m = CanberraMetric(output_transform=lambda x: (x[1], x[2]))
+        m.attach(engine, "cm")
+
+        np_y = y.numpy()
+        np_y_pred = y_pred.numpy()
+
+        canberra = DistanceMetric.get_metric("canberra")
+
+        data = list(range(y_pred.shape[0] // batch_size))
+        cm = engine.run(data, max_epochs=1).metrics["cm"]
+
+        assert canberra.pairwise([np_y_pred, np_y])[0][1] == pytest.approx(cm)
+
+    test_cases = [
+        (torch.rand(size=(100,)), torch.rand(size=(100,)), 10),
+        (torch.rand(size=(200,)), torch.rand(size=(200,)), 10),
+        (torch.rand(size=(100,)), torch.rand(size=(100,)), 20),
+        (torch.rand(size=(200,)), torch.rand(size=(200,)), 20),
+    ]
+
+    for y_pred, y, batch_size in test_cases:
+        # check multiple random inputs as random exact occurencies are rare
```

Comment:
This statement is incorrect now. Data is already generated and we run 10 times on the same data.

Reply:
I will add the `get_test_cases` part here